services/db-migration/src/main/resources/liquibase/db.changelog-2.0.xml

```diff
@@ -17,8 +17,9 @@
       <column name="status" type="VARCHAR(50)">
         <constraints nullable="false" />
       </column>
-      <column name="output_file_url" type="VARCHAR(200)"/>      
-      <column name="readable_name" type="VARCHAR(200)"/>      
+      <column name="progress" type="float" defaultValueNumeric="0"/>
```

Comment:
my rule is since none of this in prod or any other cloud environment yet we can amend the existing migrations for speed and ease. After we deploy it won't even be possible and new migrations will be needed :)